gradle-sls-packaging/src/test/groovy/com/palantir/gradle/dist/tasks/ConfigTarTaskIntegrationSpec.groovy

```diff
@@ -47,22 +48,23 @@ class ConfigTarTaskIntegrationSpec extends GradleIntegrationSpec {
         createUntarBuildFile(buildFile, "java-service", "service", "foo-service")
 
         when:
-        runTasks(':configTar', ':untar')
+        runTasksSuccessfully(':configTar', ':untar')
 
         then:
         def files = directory('dist/foo-service-0.0.1/', projectDir).list()
-        files.length == 1
+        files.length == 2
         files.contains('deployment')
         def manifest = file('dist/foo-service-0.0.1/deployment/manifest.yml', projectDir).text
         manifest.contains('service.v1')
+        file('dist/foo-service-0.0.1/service/bin/launcher-static.yml')
```

Comment:
I believe `file` will always create a new file if it doesn't exist. I'm not very familiar with groovy, but I'd assume that would evaluate to true even if a new empty file was created.
```suggestion
        fileExists('dist/foo-service-0.0.1/service/bin/launcher-static.yml')
```

Reply:
Good spot - the number of times I fall into this trap is actually incredible. Also I didn't even check for existence 🤦🏻 .